app/(auth)/login/LoginForm.tsx

```diff
@@ -31,7 +31,8 @@ export default function LoginForm(props: { returnTo?: string | string[] }) {
         const returnTo = getSafeReturnToPath(props.returnTo);
 
         if (returnTo) {
-          router.push(returnTo);
+          router.replace(returnTo);
```

Comment:
why `.replace()` instead of `.push()`

Reply:
cache revalidation

Reply:
we decided we want `.push()` - adds a history state